scep/api/api.go

```diff
@@ -333,9 +333,21 @@ func PKIOperation(ctx context.Context, req request) (Response, error) {
 
 	certRep, err := auth.SignCSR(ctx, csr, msg)
 	if err != nil {
+		// default to ERROR_INTERNAL_ERROR: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/18d8fbe8-a967-4f1c-ae50-99ca8e491d2d
+		errorCode := 0x0000054F
```

Comment:
This is very specific, I wonder if it makes more sense to use a more specific webhook type.

Reply:
I've set this to 0 in ba72710e2d11295287a9816a6d3dbd2b544ff075 for now to not overcomplicate things. A more specific webhook type can work, but having this more generic type is more suitable at the moment. Because the error code can be context specific, I make it the responsibility of the webhook receiver to handle those based on the error message (for now).